pallets/liquidity-pools/src/tests/inbound.rs

```diff
@@ -169,6 +169,8 @@ mod handle_tranche_tokens_transfer {
 				AMOUNT,
 			)
 			.unwrap();
+			let origin = EVM_DOMAIN_ADDRESS.domain().into_account();
+			assert_eq!(Tokens::balance(TRANCHE_CURRENCY, &origin), AMOUNT);
```

Comment:
I feel like here we're testing the test (testing the above line was correct)

Reply:
Definitely, removed.